v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -102,7 +138,7 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   void setPassword(String password);
 
   @TemplateParameter.Text(
-      order = 7,
+      order = 10,
```

Comment:
this should be removed ideally.

Reply:
We currently dont handle auto inference of partition column.
Will fix this when I implement that.